docs/romanisim/install.rst

```diff
@@ -17,8 +17,10 @@ operate.  See the `docs
 <https://stpsf.readthedocs.io/en/latest/installation.html#installing-the-required-data-files>`_
 for instructions on obtaining the relevant data files and pointing the
 ``STPSF_PATH`` environment variable to them.  This issue can be
-avoided by not setting the ``--stpsf`` argument, in which case
-``romanisim`` uses the GalSim modeling of the Roman PSF.
+avoided by not setting the ``--psftype=stpsf`` argument, in which case
+``romanisim`` uses the GalSim modeling of the Roman PSF. Alternatively, setting
+``--psftype=epsf`` will cause the use of CRDS EPSF reference files; see below
+for general CRDS usage.
```

Comment:
```suggestion
``STPSF_PATH`` environment variable to them.  The simplest option for getting
things to run is to use ``--psftype=galsim`` which relies on the GalSim PSF
modeling.
```

(trying to focus on the simple installation here while not getting bogged down in different kinds of PSFs)